packer/builder/azure/driver_restapi/builder.go

```diff
@@ -250,16 +255,16 @@ func (b *Builder) Run(ui packer.Ui, hook packer.Hook, cache packer.Cache) (packe
 	var err error
 	ui.Say("Preparing builder...")
 
-	ui.Message("Creating a new request manager...")
-	reqManager, err := request.NewManager(b.config.PublishSettingsPath, b.config.SubscriptionName)
+	ui.Message("Creating Azure Service Management client...")
+	b.client, err = management.ClientFromPublishSettingsFile(b.config.PublishSettingsPath, b.config.SubscriptionId)
```

Comment:
hmm I just wondered if we should make `ClientFromPublishSettingsFile` accept file contents as argument rather than path to support in-memory loading. publishsettings files are 99% of the time actually files on disk, right?